templates/layouts/html.html

```diff
@@ -1,6 +1,7 @@
 <!doctype html>
 <html>
 <head>
+  <base href="{% if NODE_ENV == "drupal" %}{{ npm_package_config_kstatic_drupalRemote}}{% else %}{{ npm_package_config_kstatic_base}}{% endif %}" />
```

Comment:
:+1: [`<base>`](http://www.w3schools.com/tags/tag_base.asp)

This could end up being used by JavaScript. Does SASS end up getting metalsmith metadata available to it? Or do relative links get resolved properly?